docs/Concepts/Privacy/Private-Transactions.md

```diff
@@ -26,18 +26,18 @@ transaction itself.
 For more information about creating and sending private transactions, see the
 [How To documentation](../../HowTo/Send-Transactions/Creating-Sending-Private-Transactions.md).
 
-## Besu and Orion keys
+## Besu and Tessera keys
 
-Besu and Orion nodes both have public/private key pairs identifying them. A Besu node sending a
-private transaction to an Orion node signs the transaction with the Besu node private key. The
+Besu and Tessera nodes both have public/private key pairs identifying them. A Besu node sending a
+private transaction to an Tessera node signs the transaction with the Besu node private key. The
```

Comment:
s/an Tessera/a Tessera

Reply:
recommend doing a find & replace for this one, there could be more

Reply:
Fixed